functions.php

```diff
@@ -373,10 +373,11 @@ function encryptLoginEntry($login_password_cleartext)
 // Get domain expiration date
 function getDomainExpirationDate($name)
 {
-
+    $nullValue = "1000-01-01";
```

Comment:
This should be 1 January 1970 00:00:00, the start of the current epoch

Reply:
I just took mySQL min date, but thats an easy change.

Preferable it should be nullable, but I have no clue of any potential issues in the codebase if the value is null. Just changed that few lines as it was stopping me from moving our docs :D

Reply:
I do not know the domain portion of this tool enough to say exactly. I think utc 0 is fine for now.